mautic/mautic
# Fix the issue with saving contact settings

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #10964 



#### Description:


After changing the Contact Settings and clicking "Save" or "Save & Close", the settings were not saved.  This PR fixes that.
#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Navigate to Configuration->Contact Settings
3. Change the options
4. Save or Save & Close
5. Navigate back to Configuration->Contact Settings again. The settings are saved
